python/examples/by-use-case/solana-usdc-yield-deposit/example.py

```diff
@@ -0,0 +1,77 @@
+import os
+import asyncio
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+
+from langchain_openai import ChatOpenAI
+from langchain.agents import AgentExecutor, create_tool_calling_agent
+from langchain_core.prompts import ChatPromptTemplate
+from solana.rpc.api import Client as SolanaClient
+from solders.keypair import Keypair
+
+from goat_adapters.langchain import get_on_chain_tools
+from goat_wallets.solana import solana
+from goat_plugins.lulo import lulo, LuloPluginOptions
+from goat_plugins.spl_token import spl_token, SplTokenPluginOptions
+from goat_plugins.spl_token.tokens import SPL_TOKENS
+
+# Initialize Solana client
+client = SolanaClient(os.getenv("SOLANA_RPC_ENDPOINT"))
+
+# Initialize regular Solana wallet
+keypair = Keypair.from_base58_string(os.getenv("SOLANA_WALLET_SEED") or "")
+wallet = solana(client, keypair)
+
+# Initialize LLM
+llm = ChatOpenAI(model="gpt-4o-mini")
+
+def main():
+    # Get the prompt template
+    prompt = ChatPromptTemplate.from_messages(
+        [
+            ("system", "You are a helpful assistant"),
+            ("placeholder", "{chat_history}"),
+            ("human", "{input}"),
+            ("placeholder", "{agent_scratchpad}"),
+        ]
+    )
+
+    # Initialize tools with Solana wallet
+    tools = get_on_chain_tools(
+        wallet=wallet,
+        plugins=[
+            lulo(LuloPluginOptions()),
```

Comment:
Do we need to pass empty options? Can it just be `lulo()`

Reply:
It is just to specify explicitly that no options are given and no defaults apply.